wls-gui-wahllokalsystem/tests/stores/wahlvorstandStore.spec.ts

```diff
@@ -111,6 +112,96 @@ describe("wahlvorstandStore.ts", () => {
     });
   });
 
+  describe("isMindestanwesenheitErreicht", () => {
+    it("should_returnFalse_when_noMitgliedExists", () => {
+      unitUnderTest.wahlvorstand.wahlvorstandsmitglieder = [];
+
+      expect(unitUnderTest.isMindestanwesenheitErreicht).toStrictEqual(false);
+    });
+
+    it("should_returnFalse_when_wahlIsNotClosedAnd2MitgliederAreAnwesend", () => {
+      const { schliessungsUhrzeitSent } = storeToRefs(useWahlbezirkStore());
+      schliessungsUhrzeitSent.value = undefined;
+
+      unitUnderTest.wahlvorstand.wahlvorstandsmitglieder = [
+        WahlvorstandsmitgliedBuilder.createMinimal()
+          .withFunktion("W")
+          .withAnwesend(true),
+        WahlvorstandsmitgliedBuilder.createMinimal()
+          .withFunktion("SB")
+          .withAnwesend(true),
+      ];
```

Comment:
Die Funktion würde ich weglassen da sie für den Test bzw. die Funktion keine Rolle spielt.

```suggestion
      unitUnderTest.wahlvorstand.wahlvorstandsmitglieder = [
        WahlvorstandsmitgliedBuilder.createMinimal()
          .withAnwesend(true),
        WahlvorstandsmitgliedBuilder.createMinimal()
          .withAnwesend(true),
      ];
```

Reply:
refactored mit [d5046d2](https://github.com/it-at-m/Wahllokalsystem/pull/1263/commits/d5046d2bee7c1071bc364b51b22e0e8407864ca9)